packages-exp/auth-exp/src/core/strategies/custom_token.test.ts

```diff
@@ -52,7 +53,7 @@ describe('core/strategies/signInWithCustomToken', () => {
   };
 
   const idTokenResponse: IdTokenResponse = {
-    idToken: 'my-id-token',
+    idToken: makeJWT({ sub: 'local-id' }),
     refreshToken: 'my-refresh-token',
     expiresIn: '1234',
     localId: serverUser.localId!,
```

Comment:
Noticed that `localId` is non-optional in `IdTokenResponse` (https://github.com/firebase/firebase-js-sdk/blob/78181760aea6c9b747807c4be07b65e801cd3a45/packages-exp/auth-exp/src/model/id_token.ts#L52) but it seems like not all responses include this field (e.g. [signInWithCustomToken](https://cloud.google.com/identity-platform/docs/reference/rest/v1/accounts/signInWithCustomToken)) - wanted to get more context on why this field is required @avolkovi / @sam-gc ? Thanks!

Reply:
Honestly it's probably an oversight